src/views/Users/ViewUser/index.jsx

```diff
@@ -445,22 +445,26 @@ function ViewUser({ isNewUser, ...props }) {
             <br />
             <br />
             <br />
-            <Typography variant="h5">Roles</Typography>
-            {roles.map(renderRole)}
-            {additionalRoles.map(renderRole)}
-            <Grid item xs className={classes.addGrid}>
-              <Grid item xs={11}>
-                <Button
-                  onClick={handleRoleAdd}
-                  className={classes.fullWidth}
-                  variant="outlined">
-                  <PlusIcon />
-                </Button>
-              </Grid>
-            </Grid>
-            <br />
-            <br />
-            <br />
+            {!isNewUser && (
+              <Fragment>
+                <Typography variant="h5">Roles</Typography>
+                {roles.map(renderRole)}
+                {additionalRoles.map(renderRole)}
+                <Grid item xs className={classes.addGrid}>
+                  <Grid item xs={11}>
+                    <Button
+                      onClick={handleRoleAdd}
+                      className={classes.fullWidth}
+                      variant="outlined">
+                      <PlusIcon />
+                    </Button>
+                  </Grid>
+                </Grid>
+                <br />
+                <br />
+                <br />
+              </Fragment>
+            )}
             <Typography variant="h5">Permissions</Typography>
```

Comment:
When adding a new user, the product restrictions autocomplete text fields has no suggestions:

![Screen Shot 2019-07-31 at 8 41 22 AM](https://user-images.githubusercontent.com/3766511/62212610-1796a080-b36f-11e9-944e-0dcfaac358bb.png) 

Is this expected?

Reply:
No, hm. I'll look into this.